components/blog/Loader.tsx

```diff
@@ -0,0 +1,27 @@
+const Loader = () => {
+	return (
+		<div className="absolute inset-0 flex items-center justify-center z-50  bg-opacity-40 bg-gray-500">
+			<output>
+				<svg
```

Comment:
Is it possible to extract this SVG into a separate file inside the `public` folder and use that directly here using the `Image` component?

LOGAF: 4/10

Reply:
[This StackOverflow answer](https://stackoverflow.com/a/38786469/4677052) gives the pros and cons of both methods.
I am inclining towards the file method for this use case. You can take the final call on this.

Reply:
done